### PR TITLE
Feat/confirmation on delete

### PR DESCRIPTION
**Pull request by adrianammt**
Small pr.

- Created a custom Confirmation Box for delete option
- Toast Message when delete is done
- Fixed MessageBox component so that it now renders messages and icons dynamically

Thanks for your review!

### CODE REVIEW

File: src/pages/Favourites.js [hunk 1]
```diff
@@ -21,7 +22,7 @@ export default function Favourites({
   return (
     <div className="ClassCard__wrapper">
       {filteredFavourites.length === 0 ? (
-        <NoFavouritesMessageBox />
+        <MessageBox icon={IoHeart} message="Add your favourite classes!" />
```
Comment (patrikglanemann) [inline src/pages/Favourites.js:25]: Nice, that is a much better to understand naming:)

**Review by patrikglanemann — COMMENTED** [review]
The confirmation works like a charm and it's good that you made it be it's own component:)

**Comment by farapara** [comment]
It looks really fine to me🙌👀. I have nothing to comment on your code. As it concerns the deployment I would suggest you could maybe consider changing the path from the fav page to the saved class instead the create page. So when the user clicks to add fav classes he goes directly to the saved ones. 😊

**Comment by adrianammt** [comment]
> It looks really fine to me🙌👀. I have nothing to comment on your code. As it concerns the deployment I would suggest you could maybe consider changing the path from the fav page to the saved class instead the create page. So when the user clicks to add fav classes he goes directly to the saved ones. 😊

**Pull request merged**